cmds/vpdbootmanager/main.go

```diff
@@ -0,0 +1,54 @@
+package main
+
+import (
+	"encoding/json"
+	"errors"
+	"flag"
+	"fmt"
+	"os"
+
+	"github.com/systemboot/systemboot/pkg/booter"
+	"github.com/systemboot/systemboot/pkg/vpd"
+)
+
+const usage = `Usage:
+boot add [netboot [dhcpv6|dhcpv4] [MAC] | localboot [grub|path [Device GUID] [Kernel Path]]]
+
+Ex.
+boot add localboot grub
+
+boot add netboot dhcpv6 AA:BB:CC:DD:EE:FF
+
+Use -h to show flag options.
+`
+
+func main() {
+	flag.Usage = func() {
+		fmt.Println(usage)
+	}
+	if err := parse(os.Args[1:]); err != nil {
```

Comment:
`parse` is not reflecting what the function does - it's actually parsing and executing the subcommand. This should be either renamed or split into parsing and execution